### ISSUE TITLE
Color Contrast on ACR Report

### ISSUE THREAD

**Issue by 508tester** [Section508Coordinators/ACRT#37]
When viewing the report on the web, the headers are black with enough contrast to the background. But when saving it as a PDF, the header becomes light grey, making it fail color contrast against the background. Is there a way to fix this so that not every PDF generated report has to be fixed for contrast?

Example: 

![Image](https://github.com/user-attachments/assets/9a55ced6-357f-43f9-954a-36a63955f41b)
